packages/uniforms/__tests__/AutoForm.tsx

```diff
@@ -27,66 +31,84 @@ describe('AutoForm', () => {
     validator.mockClear();
   });
 
-  describe('when changed', () => {
+  describe('when changes', () => {
     it('updates', () => {
-      // FIXME: AutoForm is not a valid Component.
-      const wrapper = mount<AutoForm | any>(
+      render(
         <AutoForm onChange={onChange} schema={schema} />,
+        { schema: { type: SimpleSchema2Bridge } },
```

Comment:
What is this argument? It should be used as the schema for the context, but we don't actually need seeding context here - we're rendering forms in these tests. _(Also, this is not a valid schema. At least for any value.)_

Reply:
I have changed the render function types and now the schema field is not required, If we don't pass it our component will not have a `<Context.Provider>`. I have also added the correct schema where we use context.